go-controller/pkg/ovn/ovn.go

```diff
@@ -116,20 +118,56 @@ func (oc *Controller) Run() error {
 	return nil
 }
 
+func (oc *Controller) addDefaultLogicalPort(pod *kapi.Pod) {
+	annotation := oc.addLogicalPort(pod, pod.Spec.NodeName)
+	_, isStaticIP := pod.Annotations["ovn"]
+	if !isStaticIP {
+		err := oc.kube.SetAnnotationOnPod(pod, "ovn", annotation)
+		if err != nil {
+			logrus.Errorf("Failed to set annotation on pod %s - %v", pod.Name, err)
+		}
+	}
+}
+
+func (oc *Controller) addLogicalPortsToExtraSwitches(pod *kapi.Pod) {
+	switches := oc.getNetworkNamesFromPodAnnotations(pod.Annotations)
+	if switches != nil {
+		annotations := make(map[string]string)
+		for _, logicalSwitch := range switches {
+			annotations[logicalSwitch] = oc.addLogicalPort(pod, logicalSwitch)
+		}
+
+		ovnExtraAnnotation := fmt.Sprintf("{")
+		for network, networkAnnotation := range annotations {
+			ovnExtraAnnotation = fmt.Sprintf("%s\\\"%s\\\":%s, ", ovnExtraAnnotation, network, networkAnnotation)
```

Comment:
don't you need to put between quotes also the value?

Reply:
The value is not a string, it is a map.